packages/buidler-core/src/internal/cli/cli.ts

```diff
@@ -44,6 +45,8 @@ async function main() {
   // We first accept this argument anywhere, so we know if the user wants
   // stack traces before really parsing the arguments.
   let showStackTraces = process.argv.includes("--show-stack-traces");
+  let verbose = false;
+  let config: ResolvedBuidlerConfig | undefined;
```

Comment:
Previously I had "hoisted" the config path. I could've done the same with the `analytics.enabled` value but... it was getting annoying. I don't know if there's a better way to do this (share a value between the try and catch blocks).